wordle-web/pages/game.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <v-card>
+        <v-card-title>
+            A Creative Title
```

Comment:
I feel like we were meant to come up with an actual game name here.  Could be wrong though.